opslevel/resource_opslevel_property_assignment.go

```diff
@@ -0,0 +1,124 @@
+package opslevel
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/opslevel/opslevel-go/v2023"
+)
+
+func resourcePropertyAssignment() *schema.Resource {
+	return &schema.Resource{
+		Description: "Manages properties assigned to entities (like Services)",
+		Create:      wrap(resourcePropertyAssignmentCreate),
+		Update:      wrap(resourcePropertyAssignmentUpdate),
```

Comment:
if we don't provide an `Update` method the terraform should automatically handle the the flow by doing `delete` before `create`